File: lib/antlr-2.7.5/examples/java/lexRewrite/Main.java
import java.io.*;

class Main {
	public static void main(String[] args) {
		try {
			Rewrite lexer = new Rewrite(new DataInputStream(System.in));
			lexer.mSTART(true);
			System.out.println("result Token="+lexer.getTokenObject());
		} catch(Exception e) {
			System.err.println("exception: "+e);
		}
	}
}

File: lib/antlr-2.7.5/examples/cpp/lexRewrite/Main.cpp

#include <iostream>

#include "Rewrite.hpp"

int main()
{
	ANTLR_USING_NAMESPACE(std)
	ANTLR_USING_NAMESPACE(antlr)
	try
	{
		Rewrite lexer(cin);
		lexer.mSTART(true);
		cout << "result Token=" << lexer.getTokenObject()->toString() << endl;
	}
	catch( ANTLRException& e )
	{
		cerr << "exception: " << e.getMessage() << endl;
		return -1;
	}
	catch( exception& e )
	{
		cerr << "exception: " << e.what() << endl;
		return -1;
	}
	return 0;
}

